src/store/flow/utils/importHelpers.ts

```diff
@@ -49,6 +49,64 @@ export function mergeFlowContainer(
   return existingFlowContainerStore
 }
 
+interface CreateFlowStackBlock {
+  uuid: string;
+  type: string;
+  config: {
+    flow_id: string;
+  };
+}
+
+interface CreateFlowStackFlow {
+  uuid: string;
+  blocks: CreateFlowStackBlock[];
+}
+
+interface CreateFlowStackData {
+  flows: CreateFlowStackFlow[];
+}
+
+export function createContainerFlowStack(json_data: any, flow_stack: string[]): void {
+  const flows = json_data.flows;
+  const visitedFlows: string[] = [];
+
+  function process_flow(flow: CreateFlowStackFlow): void {
+    visitedFlows.push(flow.uuid);
+    // eslint-disable-next-line no-restricted-syntax
```

Comment:
could you remind me what error do we got if we remove those `// eslint-disable-next-line no-restricted-syntax` ? I'm wondering if there is a better way to handle that

Reply:
fixed it by using array iterators